### PR TITLE
Implemented keyboard design and keys

### PR DESCRIPTION
**Pull request by flynnn231**
image

### CODE REVIEW

**Review by brunogrcsada — CHANGES_REQUESTED** [review]
Looks great Ethan! Netlify build issues must be resolved before merging. We also need to look into the positioning of black keys and the 'margin-left' calculation. Great work :)

**Review by Jsimpada — COMMENTED** [review]
The bubble in the middle doesn't have the letter for the key being pressed?

File: src/App.js [hunk 1]
```diff
@@ -32,22 +36,26 @@ function App() {
...
+        <img id="logo" src="/logo.svg" alt="Logo"/>
+        <img id="triangle" className="effects" src="/triangle.svg" alt="Visual Effect"/>
+      </div>
+
+      {/* style={{marginLeft: ((i-34)*(30)).toString() + 'px'}} */}
+
+      <div id="middle">
+       {keyboard.map((d,i) => {
+         if(d !== d.toString().toUpperCase()) return <div onMouseDown={() => play({ id: d.toString() })} id='key'><span>{d}</span></div>
+         else return <div onMouseDown={() => play({ id: d.toString() })} style={{marginLeft: ((((i-(18 * 4))*2)*30)).toString() + 'px'}} id='black_key'/>
```
Comment (melissalupu) [inline src/App.js:49]: The equation needs to be improved, the black keys are not in the correct position.🙁

**Review by elijaholan — COMMENTED** [review]
The keys are not in the right position

**Review by brunogrcsada — APPROVED** [review]
Perfect, tests are passing now!

**Pull request merged**